### PR TITLE
Fixed "unary operator expected" error in startvscode.sh

### PR DESCRIPTION
**Pull request by Wennn**
# Fixed "unary operator expected" error in startvscode.sh

- [x] You've read the [Contributor Guide](https://github.com/dotnet/aspnetcore/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/dotnet/aspnetcore/blob/main/CODE-OF-CONDUCT.md).
- [ ] You've included unit or integration tests for your change, where applicable.
- [ ] You've included inline docs for your change, where applicable.
- [ ] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.

Summary of the changes (Less than 80 chars)

## Description

Fixed the "unary operator expected" error which happens when we use the startvscode.sh without any extra arg.

### CODE REVIEW

**Review by mkArtakMSFT — APPROVED** [review]
Nice catch. Thanks @Wennn !

File: startvscode.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ if [ ! -f "$DOTNET_ROOT/dotnet" ]; then
     exit 1
 fi
 
-if [ $1 = "" ]; then
+if [[ $1 == "" ]]; then
```
Comment (dougbu) [inline startvscode.sh:23]: To more correctly handle a missing positional parameter and use the usual "is it empty❔" operator&hellip;
```suggestion
if [[ -z "${1:-}" ]]; then
```

**Review by dougbu — APPROVED** [review]
@Wennn please let me know if you prefer my suggestion. If not I'll merge this as-is later today.

File: startvscode.sh (hunk 1 above)
Comment (Wennn) [reply startvscode.sh:23]: Sorry for late response. 
Thanks for your advice and I think it should be equal at most cases to my knowledge.
Thanks again for review and merging this PR.

**Pull request merged**